mu4/palette/internal/palette/palettemodel.cpp

```diff
@@ -813,7 +813,9 @@ bool PaletteTreeModel::insertRows(int row, int count, const QModelIndex& parent)
 
         beginInsertRows(parent, row, row + count - 1);
         for (int i = 0; i < count; ++i) {
-            panel->insertCell(row, PaletteCellPtr(new PaletteCell));
+            PaletteCellPtr cell(new PaletteCell);
+            cell->id = PaletteCell::makeId();
```

Comment:
Maybe better, set id in the constructor?

Reply:
Usually, you expect from the default constructor that it will create an object with an invalid state (with id == invalid_id)